packages/telemetry/src/telemetryReporter.ts

```diff
@@ -129,23 +131,26 @@ export class TelemetryReporter extends AsyncCreatable<TelemetryOptions> {
   }
 }
 
-export class SpawnedTelemetryReporter extends AsyncCreatable<TelemetryOptions> {
+export class TelemetryReporter extends AsyncCreatable<TelemetryOptions> {
   public static SFDX_INSIGHTS_TIMEOUT = 'SFDX_INSIGHTS_TIMEOUT';
   public forkedProcess!: ChildProcess;
   private modulePath: string = MODULE_PATH;
   private options: TelemetryOptions;
   private logger!: Logger;
   private env!: Env;
+  private config!: ConfigAggregator;
 
   constructor(options: TelemetryOptions) {
     super(options);
     this.options = options;
   }
 
   public async init(): Promise<void> {
-    this.logger = await Logger.child('SpawnedTelemetry');
+    this.logger = await Logger.child('TelemetryReporter');
     this.env = this.options.env || new Env();
+    this.config = await ConfigAggregator.create({});
     this.beginLifecycle();
+    await new Promise(resolve => setTimeout(() => resolve(), 500));
```

Comment:
I still hate this =( Use https://forcedotcom.github.io/sfdx-dev-packages/kit/globals.html#sleep

Reply:
Agreed that this is a poor solution. On a positive note, I created a spike for us to figure out how we can use the APIs directly.